monai/apps/detection/networks/base_detection_network.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import abstractmethod
+from typing import Sequence, Union
+
+from torch import Tensor, nn
+
+from monai.utils import ensure_tuple_rep
+from monai.utils.module import look_up_option
+
+
+class BaseDetectionNetwork(nn.Module):
```

Comment:
Hi @Can-Zhao this kind of `BaseDetectionNetwork` generic abstraction will need more discussions, could you please remove it for now,